payment_paypal/indico_payment_paypal/util.py

```diff
@@ -7,11 +7,10 @@
 
 import re
 
-from wtforms import ValidationError
-
 from indico.util.string import is_valid_mail
 
 from indico_payment_paypal import _
+from wtforms import ValidationError
```

Comment:
again here, isort should fix this..

Reply:
I did an isort to all three python files changed before committing. Anyway, it's not the latest release, so let's wait until I can finish them all.